pkgs/applications/misc/msglint/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "msglint";
+  version = "1.04";
+
+  sourceRoot = ".";
+
+  src = fetchurl {
+    url = "https://tools.ietf.org/tools/msglint/msglint-${version}.tar.gz";
+    sha256 = "1yasj6nijip3z86m526bsssd5vgqrx233azvpz1c0z564wngvzbm";
+  };
+
+  patches = [ ./fixfree.patch ];
```

Comment:
```suggestion
  patches = [
    # What does this patch fix?
    ./fixfree.patch
  ];
```

Reply:
IIRC, this fixes potential segfaults for files passed as arguments. But yes, some describing comment would be good.